components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql

```diff
@@ -1832,7 +1832,7 @@ CREATE TABLE IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```

Comment:
This should not have a NULL default value. Please pick appropriate default value 0 or 1.

Reply:
resolved